.github/workflows/check.yaml

```
@@ -27,13 +27,12 @@ jobs:
     with:
       additional-r-cmd-check-params: --as-cran
   coverage:
-    if: github.event_name == 'pull_request'
-    name: Coverage 📔 
+    name: Coverage 📔
```

```suggestion
    if: github.event_name != 'push'
    name: Coverage 📔
```
What about `if` condition?

---

Running coverage also for push to `main` makes it possible to generate e.g. coverage badges for `main` in case we'd like to use them in the future.
But we can also disable that for now.